constitution.md

```diff
@@ -0,0 +1,196 @@
+# Constitution of the Unix Users Group
+
+October 25, 1999  
+Authored by David Wilburn,  
+Amended, May 1, 2003.  
+Amended, April 1, 2019.  
+
+## Article I - Name of Organization
+
+This organization shall be called the James Madison University Unix Users
+Group. The Unix Users Group is also referred to as the UUG.
+
+## Article II - Purpose of Organization
```

Comment:
This need to be updated to reflect our current purpose. Are we just Unix-like or all open-source?

Reply:
I have always understood "Unix users" in a fairly broad sense. In particular, I have always understood it to definitely NOT be in the copyright sense, so Linux and MacOS (as a descendant of BSD) are definitely included (even if not always open source). I guess I've generally understood it as a "computer systems enthusiast" club for anyone who is interested in how to use low-level features of their computers, be it the OS, the shell, the windowing system, networking, etc. Basically all the systems-y stuff that's not already covered by CDC and Forensics, but with less emphasis on programming than the ACM competitive programming club. The name of the club to me is really just a nod to the historical significance of Unix and its influence on all areas of systems since its creation.